frontend/templates/troubleshooting/index.tsx

```diff
@@ -22,36 +22,37 @@ const Wiki: NextPageWithLayout<{
    layoutProps: DefaultLayoutProps;
 }> = ({ wikiData }) => {
    return (
+      <Flex direction="column" alignItems="center" width="100%" fontSize="16px">
          <NavBar />
-      <Flex
-         direction="row"
-         justifyContent="center"
-         width="100%"
-         fontSize="16px"
-      >
-         <Flex padding="0px 32px 32px" gap="48px" maxW="1280px" id="wrapper">
-            <Flex gap="16px" direction="column" id="main">
-               <Head>
-                  <meta name="robots" content="noindex" />
-               </Head>
-               <Heading as="h1">{wikiData.title}</Heading>
-               {wikiData.introduction.map((intro) => (
-                  <IntroductionSection key={intro.heading} intro={intro} />
-               ))}
-               {wikiData.solutions.map((solution, index) => (
-                  <SolutionCard
-                     key={solution.heading}
-                     index={index + 1}
-                     solution={solution}
-                  />
-               ))}
-               {wikiData.conclusion.map((conclusion) => (
-                  <ConclusionSection
-                     key={conclusion.heading}
-                     conclusion={conclusion}
-                  />
-               ))}
-            </Flex>
+         <Flex
+            padding="0px 32px 32px"
```

Comment:
I am little bit confused as to why you are setting the padding for right and left here but then overriding it directly with `paddingLeft` and `paddingRight`?

Why not set them here `padding="0px 48px 32px"`

Reply:
Neither do I! I must have missed that there was already padding on this. I'll combine them and push a commit.

Reply:
Should be fixed in d110a7a1ddf9bebac1a09429479772f626db6dbf.

Reply:
To confirm, we do want `32px` for left and right padding instead of `48px`?

Reply:
Yes: looking back at the mockup, I'm pretty sure that's the correct value.